vkFFT/vkFFT.h

```diff
@@ -1250,6 +1266,10 @@ static inline VkFFTResult VkFMAReal(VkFFTSpecializationConstantsLayout* sc, cons
 }
 static inline VkFFTResult VkMulComplex(VkFFTSpecializationConstantsLayout* sc, const char* out, const char* in_1, const char* in_2, const char* temp) {
 	VkFFTResult res = VKFFT_SUCCESS;
+#if(VKFFT_BACKEND==2)
+	sc->tempLen = sprintf(sc->tempStr, "\
+	%s = %s * %s.x + %s(-%s.y, %s.x) * %s.y;\n", out, in_1, in_2, sc->vecType, in_1, in_1, in_2);
+#else
```

Comment:
This one I'm worried about a bit more, is the logic too simplistic here, does this look ok?

Reply:
I am also not sure if packed math actually works fully in this case and how the vector initialization inside the math is handled - I don't use that to make code easier. It is also missing the implementation for the other mode of functioning of VkMulComplex using temp value - when out can be the same as one of inputs.

Reply:
> I am also not sure if packed math actually works fully in this case and how the vector initialization inside the math is handled

The instruction have quite extensive modifiers so llvm seems to be able to handle this very well: [godbolt example](https://godbolt.org/z/d8xz19Me7)

>  It is also missing the implementation for the other mode of functioning of VkMulComplex using temp value - when out can be the same as one of inputs.

I'll fix that up.

Reply:
I see, this one is looking good in this one-line way.